docs/layouts/shortcodes/generated/rest_v1_dispatcher.html

```diff
@@ -2335,6 +2335,9 @@
     "state_backend" : {
       "type" : "string"
     },
+    "state_changelog_enabled" : {
+      "type" : "boolean"
+    },
```

Comment:
Is this actually related to this PR? Or #20103?

Reply:
Hi @rkhachatryan , here is [#20103](https://github.com/apache/flink/pull/20103) .